examples/postgresql-backup-provided-service-account/main.tf

```diff
@@ -45,20 +45,29 @@ resource "google_storage_bucket" "backup" {
   project       = var.project_id
 }
 
+resource "google_monitoring_notification_channel" "email" {
+  display_name = "Test email notification channel"
+  type         = "email"
+  labels = {
+    email_address = "test@acme.com"
+  }
+}
+
 module "backup" {
-  source  = "terraform-google-modules/sql-db/google//modules/backup"
-  version = "~> 18.0"
+  source = "../../modules/backup"
```

Comment:
@imrannayer If there is better way to handle this before new release let me know

Reply:
@ps-occrp keep the `source` as it was. Our integration test replace it before running the test.